### DISCUSSION TITLE
Changing color in different severity and states

### DISCUSSION THREAD

**Discussion by MinaAgrez** [primefaces/community#2232] [Figma UI Kit] [answered]
Hello

We are using PrimeOne-2.1.1 components in Figma and we are wondering how to change colors in different severity and states.

For example, we are using one button which has different colors depending on the state. We would like to change default color in the Hover state (the Idle state is working when we want to change it...). We tried to change it by choosing state -> Hover

![image](https://github.com/primefaces/community/assets/174302875/5a8a8e32-4463-48ae-b55b-c4e7e31e7de2)

And select a different color:

![image](https://github.com/primefaces/community/assets/174302875/efbdbc71-c69c-4ce6-8028-b5695281d6a3)

After selecting the color, we changed the state to idle, but the color in the hover state didn't change when we presented our prototype.

We also tried to set the colors in the library, where we have all the components, and the result was the same.

Do you have any advice? Thank you in advance!

**Comment by onursenture** [comment, marked as answer]
Hello, for a color change, the ideal way in your case is to make the changes in the library's main file and then publish the library again for the changes to take effect in consumer files. 

While making the changes, make sure you edit the main component and not the instance by using 'Go to main component.' 

Additionally, after selecting the button component, you can see which variants are Hover by choosing Hover from the State section in the Properties panel on the right.

<img width="560" alt="resim" src="https://github.com/primefaces/community/assets/244957/a0faee68-4d00-48ab-a10b-99ea4543ba5d">

If you encounter any issues while trying this, please let us know so we can assist you.

**Reply by MinaAgrez** [reply]
OMG it is working... Than you very muck - the publish step was missing :)
And I can see you can even get to update all existing buttons at once...

Thank you again
